File: proyecto_python/paquete05/ciclismo.py
def registrar_participante_ciclismo():
    nombre = input("Ingrese el nombre del participante: ")
    edad = int(input("Ingrese la edad del participante: "))
    tipo_ciclismo = input("Ingrese el tipo de ciclismo (montaña, ruta, urbano): ")
    ciudad = input("Ingrese la ciudad: ")
    marca_bicicleta = input("Ingrese la marca de bicicleta preferida: ")

    cadena = f"""
    Resumen de registro - Ciclismo:
    Nombre: {nombre}
    Edad: {edad}
    Tipo de ciclismo: {tipo_ciclismo}
    Ciudad: {ciudad}
    Marca de bicicleta preferida: {marca_bicicleta}
    """

    return cadena

File: proyecto_java/Problema01/src/paquete05/Ciclismo.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package paquete05;

import java.util.Scanner;

/**
 *
 * @author Core i5-Pro
 */
public class Ciclismo {
    public static String registrarParticipanteCiclismo() {
        Scanner entrada = new Scanner(System.in);
        String mensaje;
        String nombre;
        int edad;
        String tipociCiclismo;
        String marcaBicicleta;
        System.out.print("Ingrese el nombre del participante: ");
        nombre = entrada.nextLine();
        System.out.print("Ingrese la edad del participante: ");
        edad = entrada.nextInt();
        entrada.nextLine();
        System.out.print("Ingrese el tipo de ciclismo (montaña,ruta,urbano): ");
        String tipoCiclismo = entrada.nextLine();
        System.out.print("Ingrese la ciudad: ");
        String ciudad = entrada.nextLine();
        System.out.print("Ingrese la marca de bicicleta preferida: ");
        marcaBicicleta = entrada.nextLine();
        
        String cadena= "Resumen de registro - Ciclismo:\n" +
               "Nombre: " + nombre + "\n" +
               "Edad: " + edad + "\n" +
               "Tipo de ciclismo: " + tipoCiclismo + "\n" +
               "Ciudad: " + ciudad + "\n" +
               "Marca de bicicleta preferida: " + marcaBicicleta + "\n";
        mensaje= cadena;

        return mensaje;
    }
}
